tools/src/test/java/org/apache/kafka/tools/consumer/group/ResetConsumerGroupOffsetTest.java

```diff
@@ -659,6 +663,26 @@ public void testResetWithUnrecognizedNewConsumerOption(ClusterInstance cluster)
         assertThrows(OptionException.class, () -> getConsumerGroupService(cgcArgs));
     }
 
+    @Timeout(60)
```

Comment:
Nit: No need to add `@Timeout` here, `@ClusterTest` already includes a timeout setting.

Reply:
@m1a2st  Thanks very much, the `@Timeout` is redundant, and I have remove it, PTAL